### PR TITLE
Remove Blender exporter, add 'Loading 3D models' guide.

### PR DESCRIPTION
**Pull request by donmccurdy**
Per discussion in https://github.com/mrdoob/three.js/issues/12903, this PR removes the Blender exporter for the three.js JSON format, to focus on improving support for other workflows. The Three.js JSON format is still fully supported for use with [Object3D.toJSON](https://threejs.org/docs/#api/core/Object3D.toJSON), the [Editor](https://threejs.org/editor/), [THREE.ObjectLoader](https://threejs.org/docs/#api/loaders/ObjectLoader), [THREE.JSONLoader](https://threejs.org/docs/#api/loaders/JSONLoader), and [converters](https://github.com/mrdoob/three.js/tree/dev/utils/converters). Archived versions of the exporter are no longer maintained, but will continue to work with loaders indefinitely.

I'd like to be clear about the rationale for this change — the Blender exporter has provided a viable workflow since 2010 ([initial commit](https://github.com/mrdoob/three.js/commit/cb76523b31ec3bc02a0263e92884cac4c39cc5da#diff-12f78f1697220d95d13b9959fcf4df7c)) and includes contributions from many authors that have been extraordinarily valuable to the three.js library in that time. An enthusiastic _thank you!_ to all contributors.

However, there are many [open issues](https://github.com/mrdoob/three.js/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22Blender+Exporter%22+) today, and resources to maintain the exporter are limited. And unfortunately, its perceived status as the "official three.js exporter" leads many new users to try it first, even as other formats like glTF and FBX have become increasingly reliable. By encouraging developers and artists to choose the most stable workflows, results will be better for more people today. And by focusing our own maintenance efforts, 3D models on the web will get better faster in the long term.

In its place, and as a starting point for new users, I've added a [Loading 3D Models](https://threejs.org/docs/#manual/introduction/Loading-3D-models) section in the documentation. The documentation recommends glTF where possible, and provides troubleshooting tips that are applicable to any format.

### CODE REVIEW

**Comment by looeee** [comment]
The guide looks good! Thanks for adding this. 

[Live version](https://rawgit.com/donmccurdy/three.js/44aaa9c7bc57938427b9fc051b266433971d37b3/docs/#manual/introduction/Loading-3D-models)

> TODO: Do we recommend model-related questions go to GitHub, Stack Overflow, or the Discourse forum?

I would say the forum over StackOverflow - if we are recommending somewhere else, then why have an official forum in the first place?

**Comment by Mugen87** [comment]
Yeah, the forum seems a good place for those questions.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Bye bye Blender exporter  👋

**Comment by Mugen87** [comment]
Happy

File: docs/manual/introduction/Loading-3D-models.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    If you've gone through the troubleshooting process above and your model
+    still isn't working, the right approach to asking for help will get you to
+    a solution faster. Whenever possible, include your model (or a simpler
+    model with the same problem) in any formats you have available. Include
+    enough information for someone else to reproduce the issue quickly —
+    ideally, a live demo.
+  </p>
+
+  <p>
+    TODO: Do we recommend model-related questions go to GitHub, Stack Overflow,
```
Comment (Mugen87) [inline docs/manual/introduction/Loading-3D-models.html]: We should remove this TODO  on occasion^^. Model-related questions should go to stackoverflow and the forum.

Comment (donmccurdy) [reply docs/manual/introduction/Loading-3D-models.html]: Opened #14179

**Comment by mrdoob** [comment]
> doctor, my leg hurts - no problem, we will remove it - but can't I just keep it? - no, you complain about pain too much.

The doctor gave the patient a prosthesis and the patient was not taking care of it 😛

**Comment by oewstyle** [comment]
Hi guys, the link is wrong or file as moved (https://cdn.rawgit.com/donmccurdy/three.js/deprecate-blender-exporter/docs/#manual/introduction/Loading-3D-models) 404: Not Found. Can someone gives us the good one ? Cheers.
Oh ! and by the way Three js is the best thing ever ! i'm waiting for such a thing since 2009. Congratulation to everyone who work hard on this and i hope to give a hand if i can ASAP .

**Comment by Mugen87** [comment]
https://threejs.org/docs/index.html#manual/introduction/Loading-3D-models

**Comment by DefecateRainbows** [comment]
This hurts. I spent days trying to get the crappy old exporter to work, and a few days after I actually get a model in without some sort of bug... this happens... I'm probably wrong, but now I feel like I will have to go through the same trouble again with glTF...

At least I will be one of the last to have to struggle with the blender exporter.
This is a great lib, thanks for all of the work you guys put into it.
Goodbye Blender Exporter...

**Comment by donmccurdy** [comment]
Sorry that the timing bit you here — if you're trying glTF (or another format) feel free to reach out in the three.js forums, the [glTF gitter channel](https://gitter.im/KhronosGroup/glTF), or GitHub issues for the various export tools.

**Pull request merged**